src/main/java/com/gooddata/md/maintenance/PartialMdExport.java

```diff
@@ -0,0 +1,55 @@
+package com.gooddata.md.maintenance;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+import org.springframework.web.util.UriTemplate;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Partial metadata export configuration structure
+ */
+@JsonTypeName("partialMDExport")
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+public class PartialMdExport {
```

Comment:
Yes, after other code review changes, this class became as the input parameter of the public API method  `partialExport`.